src/flask/templates/checklist_page.html

```diff
@@ -0,0 +1,208 @@
+            border-radius: 10px;
+            background-color: white;
+        }
+
+        .radio-item {
+            display: flex;
+            align-items: center;
+            margin: 10px;
+            width: 100%;
+            justify-content: space-between;
+            word-wrap: break-word;
+        }
+
+        .radio-item input[type="checkbox"] {
+            margin-right: 10px;
+            transform: scale(1.5);
+        }
+
+        .radio-item label {
+            flex-grow: 1;
+            padding: 10px 20px;
+            font-size: 18px;
+            background-color: #ddd;
+            border: 2px solid #ccc;
+            border-radius: 5px;
+            cursor: text;
+            transition: background-color 0.3s, border-color 0.3s;
+            margin-right: 10px;
+            white-space: pre-wrap; /* Ensures the text wraps to the next line */
+            overflow-wrap: break-word; /* Allows breaking of words to the next line if too long */
+        }
+
+        .radio-item input[type="checkbox"]:checked + label {
+            text-decoration: line-through;
+            color: #999;
+            background-color: #f0f0f0;
+            border-color: #ccc;
+        }
+
+        .radio-item label:focus {
+            outline: none;
+            border-color: #4a90e2;
+        }
+
+        .delete-button {
+            background-color: #ff6b6b;
+            color: white;
+            border: none;
+            border-radius: 5px;
+            cursor: pointer;
+            padding: 5px 10px;
+            font-size: 16px;
+            transition: background-color 0.3s;
+        }
+
+        .delete-button:hover {
+            background-color: #ff4949;
+        }
+
+        .add-button {
+            position: fixed;
+            bottom: 20px;
+            right: 20px;
+            padding: 10px 20px;
+            font-size: 18px;
+            background-color: #4a90e2;
+            color: white;
+            border: none;
+            border-radius: 5px;
+            cursor: pointer;
+            transition: background-color 0.3s;
+        }
+
+        .add-button:hover {
+            background-color: #357abd;
+        }
+    </style>
+</head>
+<body>
+    <div class="container">
+        <div class="header">
+            <h1>DAILY TO DO LIST</h1>
+        </div>
+        <div class="radio-group" id="radio-group">
+            <div class="radio-item" id="radio1">
+                <input type="checkbox" onchange="toggleStrikethrough(this)">
+                <label contenteditable="true" onclick="event.stopPropagation()">Start typing...</label>
+                <button class="delete-button" onclick="deleteItem(this)">X</button>
+            </div>
+        </div>
+        <button class="add-button" onclick="addRadioButton()">Add New Item</button>
+    </div>
+
+    <script>
+        const sendRequest = (endpoint, payload) => {
+            // All endpoints follow the POST method and return the same response
```

Comment:
Should we move the Javascript into a separate file to easily unit test it?

Reply:
Agreed but lets do this in a separate PR